db/schema.rb

```diff
@@ -268,6 +269,7 @@
     t.string   "uid"
     t.integer  "cove_id"
     t.string   "full_name"
+    t.string   "rep_group"
```

Comment:
This should not be necessary, as it should be handled by the `acts_as_taggable_on` gem